source/app/util/environment-check.ts

```diff
@@ -20,7 +20,7 @@ import isTraySupported from './is-tray-supported'
 import commandExists from 'command-exists'
 
 export default async function environmentCheck (): Promise<void> {
-  global.log.info('Performing environment check ...')
+  console.log('Performing environment check ...')
```

Comment:
Sorry for my ignorance, but why it is logged to the console in this file, instead of to the log provider?

Reply:
Ah wait, it's because the environment check is run very early before the log provider is even initialized, right?

Reply:
Yes but I agree it's not good style! But I may have an idea of how to solve that …

Reply:
Amazing how easy it can be once you remove any unsafe code from the constructors!